utils/supabase/supabase.ts

```
@@ -0,0 +1,65 @@
+}
+
+/**
+ * Creates and returns a Supabase client configured for use within Next.js route handlers.
+ *
+ * The client is initialized with environment variables and integrates with the Next.js cookie store to manage authentication and session cookies.
+ */
+export async function createRouteHandlerClient() {
+  const cookieStore = await cookies()
+
+  return createServerClient(
+    process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)